data/migrations/20210125173715_add_votes-remaining_faceoff-ballots_to_child_votes-casted_to_faceoffs.js

```diff
@@ -0,0 +1,24 @@
+
+exports.up = function(knex) {
+    return knex.schema
+    .table('Children', table =>{
+        table.specificType('Ballots', 'integer ARRAY').defaultTo(knex.raw('ARRAY[]::integer[]'));
+        table.integer('VotesRemaining').defaultTo(3).notNullable();
+    })
+    .table('Faceoffs', table =>{
+        table.integer('VotesCasted').defaultTo(0).notNullable();
+    })
+};
+
+
+
+exports.down = function(knex) {
+    return knex.schema
+    .table('Faceoffs', table =>{
+        table.dropColumn('VotesCasted');
+    })
+    .table('Children',table =>{
+        table.dropColumn('Ballots');
+        table.dropColumn('VotesRemaining');
```

Comment:
Ooh, I didn't realize that you could drop the columns in the same order they were added. I knew the tables had to be reversed & just assumed the columns were the same. Interesting